service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/handlers/FurtherEvidenceUploadedEventHandlerEmailTemplateTest.java

```diff
@@ -43,69 +53,170 @@
     private static final LocalDateTime HEARING_DATE = LocalDateTime.of(2021, 2, 22, 0, 0, 0).plusMonths(3);
     private static final String RESPONDENT_LAST_NAME = "Smith";
     private static final String CHILD_LAST_NAME = "Jones";
+    private static final UUID REPRESENTATIVE_UUID = UUID.randomUUID();
+    private static final CaseData caseData = buildCaseData();
+    private static final CaseData caseDataBefore = buildCaseDataBefore();
```

Comment:
These look like they should be `CASE_DATA` and `CASE_DATA_BEFORE` for consistency